spec/models/rook_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    before(:all) do
```

Comment:
You don't need this before block because your tests are simple enough to not require any database interaction. I would recommend using a `let` block. See `king_spec.rb` for an example.